src/EVault/shared/types/LTVConfig.sol

```diff
@@ -7,16 +7,18 @@ import {ConfigAmount} from "./Types.sol";
 /// @title LTVConfig
 /// @notice This packed struct is used to store LTV configuration of a collateral
 struct LTVConfig {
```

Comment:
I know that names become longer, but I think that it would be cleaner to have
- `liquidationLTV` → `targetLiquidationLTV`, to be explicit that this is not the current LTV (or at least it depends on ramp) and to make a statement that it's different compared to how `borrowLTV` is returned (it's always static value)
- `targetTimestamp` → `targetLiquidationTimestamp` to be clear that it's not related to `borrowLTV`
-  `rampDuration` → `targetLiquidationRampDuration` to be clear that it's not related to `borrowLTV`

These are just subjective (and probably nitpick) suggestions. So take them with a grain of salt and feel free to acknowledge them or modify the nomenclature suggested.

Reply:
Thanks, we'll consider the changes